org.wso2.codequalitymatrices/src/test/java/com/wso2/code/quality/metrics/Token.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.wso2.code.quality.metrics;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+/**
+ * This class is used for providing github and WSO2 PMT access tokens
+ *
+ * @since 1.0.0
+ */
+public class Token {
+    final String pmtToken = "tQU5vxzrGeBpLMQuwOsJW_fyYLYa";
```

Comment:
Can't this be extracted out to the properties file? Looks like a hard coded config value.